docs/js/search.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+    function displaySearchResults(results, store) {
+      var searchResults = document.getElementById('search-results');
+      var baseURL = "/codewind";
+  
+      if (results.length) { // Are there any results?
+        var appendString = '';
+  
+        for (var i = 0; i < results.length; i++) {  // Iterate over the results
+          var item = store[results[i].ref];
+          appendString += `
+          <div class="card">
+            <div class="card-body">
+              <h5 class="card-title">${item.title}</h5>
+              <p class="card-text">${item.content.substring(0, 150)}</p>

```

Comment:
I'd prefer using blog summaries here, as opposed to content substrings. At the moment we're not handling new lines, titles are being repeated, and we're stopping mid sentence on some. 

We could make all blog/news/docs/guides pages have a ‘description’ and use that. From what I can see that’s just changing `blog_description` / `guide_description` to description then adjusting everywhere we use them.